newIDE/app/src/ProjectsStorage/ProjectContentChecker.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import { showErrorBox } from '../UI/Messages/MessageBox';
+import { t } from '@lingui/macro';
+import { I18n } from '@lingui/react';
+import { type I18n as I18nType } from '@lingui/core';
+
+const gd = global.gd;
+
+export default function verifyProjectContent(i18n, content) {
```

Comment:
You have to type `verifyProjectContent`: 
`verifyProjectContent(i18n: I18nType, content: Object): boolean {`

otherwise flow will yell at you:
![image](https://user-images.githubusercontent.com/1280130/67319419-d7c1fe00-f504-11e9-8e48-1141b57f8c3b.png)

(usually `Object` is not a great typing, but it's actually the proper thing to use here).